tests/test_dataflow/core/test_linearize.py

```diff
@@ -139,3 +140,11 @@ def test_meta():
         ")",
         ")",
     ]
+
+
+def test_meta_to_canonical():
+    s = """( Yield ( Execute ( ReviseConstraint ( refer ( ^ ( Dynamic ) roleConstraint ( Path.apply "output" ) ) ) ( ^ ( Event ) ConstraintTypeIntension ) ( Event.showAs_? ( ?= ( ShowAsStatus.OutOfOffice ) ) ) ) ) )"""
```

Comment:
It was my understanding that `"output"` would actually look like `" output "` in the linearized output, and that this was important to getting good numbers with seq2seq. @hao-fang is that true?

Reply:
maybe `"output"` is not the best example for it mattering... more important are strings that were copied from the utterance, like `# ( PersonName " Dan Schoffel " )`

Reply:
Yeah, you are right.
But if there is no space in the double quote for `Path.apply` then it may be okay?
The `# ( PersonName " Dan Schoffel " )` example may be handled by the `sexp_formatter` and `sexp_deformatter`.

Reply:
Yeah, sexp_deformatter handles it. 

I'll add a test case. This is a copy-paste from Elias so I don't know why the quotes look that.